t/43-cli-api.t

```diff
@@ -110,6 +110,10 @@ subtest 'API' => sub {
     throws_ok { $api->run(@auth) } qr/Usage: openqa-cli api/, 'usage';
     is $api->apikey, 'ARTHURKEY01', 'apikey';
     is $api->apisecret, 'EXCALIBUR', 'apisecret';
+
+    $api = OpenQA::CLI::api->new;
+    throws_ok { $api->run('--apibase', '') } qr/Usage: openqa-cli api/, 'usage';
```

Comment:
This looks very strange to me... but I guess this is a regression and this time we have proper test coverage.

Reply:
these tests are missing actual parameters, that's why they fail. but still it can be tested if the apibase parameter was set correctly